src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSBuildReferenceTaxonomyUtils.java

```diff
@@ -18,7 +18,7 @@
 public final class PSBuildReferenceTaxonomyUtils {
 
     protected final static Logger logger = LogManager.getLogger(PSBuildReferenceTaxonomyUtils.class);
-    private final static String ROOT_ID = "1"; //NCBI root node taxonomic id
+    final static String ROOT_ID = "1"; //NCBI root node taxonomic id
```

Comment:
Are taxon ID always integers? and if so, why not type them like such?

Reply:
Yes especially since Java strings are so memory hungry. I have refactored all the code this way.